src/vaadin-grid-scroll-mixin.html

```diff
@@ -77,6 +78,31 @@
       return this.$.table.scrollTop;
     }
 
+    constructor() {
+      super();
+      this._scrollLineHeight = this._getScrollLineHeight();
+      this._scrollPageHeight = this._scrollLineHeight;
```

Comment:
I thought one line default would be better than nothing in case for some reason the user would run into this default (if the value hasn't been updated yet when a scroll happens) and having scroll by lines configured. If it was undefined I think scrolling wouldn't work at all.

Do you have any ideas for a better default this soon (in the constructor)?

Reply:
Maybe this line could be left out? I don't see a case where a user would be able to scroll the grid before this has a proper value (assuming `_viewportHeight` gets initialized right away)

Reply:
Ok. This is now removed.